src/shippingservice/src/main.rs

```diff
@@ -82,7 +91,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .await;
 
     init_logger()?;
-    init_tracer()?;
+    let tracer = init_tracer()?;
+    init_reqwest_tracing(tracer)?;
```

Comment:
same as other comment -- I think this one makes a little less sense with the `?` but still think this reads better as 

```
init_reqest_tracing(init_tracer()?)?;
```

my 2c.